haskell/c2hs.bzl

```diff
@@ -59,23 +61,36 @@ def _c2hs_library_impl(ctx):
         (version_macro_headers, version_macro_flags) = version_macro_includes(dep_info)
         args.add_all(["-C" + x for x in version_macro_flags])
 
+    (inputs, input_manifests) = ctx.resolve_tools(tools = [c2hs])
+
     hs.actions.run_shell(
         inputs = depset(transitive = [
             depset(cc.hdrs),
-            depset([hs.tools.ghc, c2hs, chs_file]),
+            depset([chs_file]),
             depset(dep_chi_files),
             depset(cc.files),
             set.to_depset(version_macro_headers),
+            inputs,
         ]),
+        input_manifests = input_manifests,
+        tools = [hs.tools.ghc, c2hs_exe],
         outputs = [hs_file, chi_file],
-        command = """
+        command =
+            # cpp (called via c2hs) gets very unhappy if the mingw bin dir is
+            # not in PATH so we add it to PATH explicitely.
+            (
+                """
+        export PATH=$PATH:{mingw-bin}
+        """.format(cc = paths.dirname(cc.tools.cc)) if hs.toolchain.is_windows else ""
```

Comment:
`cc` doesn't match the place holder `{mingw-bin}`.

Reply:
Good catch, fixed!